lunar_lander/main.py

```diff
@@ -1,12 +1,30 @@
 import gym
 import time
+from typing import Dict, Tuple
+import numpy as np
+
 
 env = gym.make("LunarLander-v2")
-actions = {"no_action": 0, "left_engine": 1, "main_engine": 2, "right_engine": 3}
+StateType = np.ndarray
+actions: Dict[str, int] = {
```

Comment:
🔧 If we plan on only using python>=3.9 I think we can use just `dict` and `tuple` instead of `typing.Dict` and `typing.Tuple`, e.i.,
`actions: dict[str, int] = ...`

https://stackoverflow.com/questions/70449405/when-why-use-types-from-typing-module-for-type-hints